dallinger/prolific.py

```diff
@@ -146,22 +203,91 @@ def draft_study(
     ) -> dict:
         """Create a draft Study on Prolific, and return its properties."""
 
+        config = get_config()
+        config.load()
+
+        # We won't specify the project id if this is True when we call the Prolific API endpoint.  Which will instruct
+        # Prolific to create the study in the default project.
+        use_prolific_default_project = True
+
+        # These two try blocks are nested to make implementing different behaviors for each exception easier, should
```

Comment:
Stylistically this seems unwieldy to me. I would think it better to add a couple of auxiliary functions here, so you can have a structure like:

```py
def published_study(...): 
    config = get_config()
    config.load()

    workspace_id = self._get_workspace_id(config)
    project_id = self._get_project_id(config)
```

Reply:
Because we ran out of time we agreed to just merge this without resolving this suggestion